packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -209,11 +208,7 @@ class _SnapshotViewState extends State<SnapshotView> with AutoDisposeMixin {
 
     addAutoDisposeListener(controller.snapshotJsonFile);
 
-    addAutoDisposeListener(controller.processingNotifier, () {
-      setState(() {
-        processing = controller.processingNotifier.value;
-      });
-    });
+    addAutoDisposeListener(controller.processingNotifier);
```

Comment:
you don't need to listen to the notifier if you use a value listenable builder. If you don't specify a listener in `addAutoDisposeListener`, `setState` is still called on the whole widget. So you can remove this line